package.json

```diff
@@ -87,6 +87,9 @@
     "webpack-dev-server": "^5.2.0",
     "winston-aws-cloudwatch": "^3.0.0"
   },
+  "overrides": {
```

Comment:
Do we still need an override?

Reply:
Even with the upgrade to eslint 9 and the override, I'm still seeing `cross-spawn 7.0.3` showing up in the security scan. `webpack-cli` is using the newest version, and it depends on `cross-spawn ^7.0.3`, but `7.0.6` should be satisfying that.

Reply:
Seems to likely be an issue with Grype?
```
❯ npm ls cross-spawn          
crc-pdf-generator@1.0.0 /home/comitche/Projects/pdf-generator
├─┬ eslint@9.26.0
│ ├─┬ @modelcontextprotocol/sdk@1.11.0
│ │ └── cross-spawn@7.0.6 deduped
│ └── cross-spawn@7.0.6
├─┬ glob@10.4.5
│ └─┬ foreground-child@3.2.0
│   └── cross-spawn@7.0.6 deduped
├─┬ jest@29.7.0
│ └─┬ @jest/core@29.7.0
│   └─┬ jest-changed-files@29.7.0
│     └─┬ execa@5.1.1
│       └── cross-spawn@7.0.6 deduped
└─┬ webpack-cli@6.0.1
  └── cross-spawn@7.0.6 deduped
  ```

Reply:
Yeah I think this is good.